scripting/include/srccoop/manager.inc

```diff
@@ -555,7 +555,25 @@ methodmap CoopManager
 	{
 		return (ccl == CCL_MAPS);
 	}
-	
+
+	/**
+	 * Finds an index of the team to enforce.
+	 * Must be called after teams had been created.
+	 * Returns -1 on failure / if teams are not to be enforced.
+	 */
+	public static int GetEnforcedTeam()
+	{
+		char szTeam[MAX_TEAMNAME];
+		int iTeam;
+
+		g_pConvarCoopTeam.GetString(szTeam, sizeof(szTeam));
+		if (!StringToIntEx(szTeam, iTeam))
+		{
+			iTeam = FindTeamByName(szTeam);
```

Comment:
I noticed the description of `g_pConvarCoopTeam` says that the ConVar takes a partial team name or team number. Does `FindTeamByName` support team numbers as the string parameter? https://sm.alliedmods.net/new-api/sdktools_stocks/FindTeamByName

Reply:
It's only called when StringToIntEx fails